package.json

```diff
@@ -82,19 +86,23 @@
     "fixturify": "^1.2.0",
     "istanbul": "^0.4.3",
     "loader.js": "^4.7.0",
+    "lodash.mergewith": "^4.6.2",
     "mocha": "^8.1.0",
     "mocha-eslint": "^6.0.0",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
+    "qunit": "^2.14.0",
     "release-it": "^14.0.2",
     "release-it-lerna-changelog": "^2.4.0",
     "rsvp": "^4.8.5",
     "sinon": "^9.1.0",
     "testdouble": "^3.16.1"
   },
   "peerDependencies": {
+    "@ember/test-helpers": "*",
     "ember-mocha": "*",
-    "ember-qunit": "*"
+    "ember-qunit": "*",
+    "qunit": "*"
```

Comment:
Can you add this to peerDependenciesMeta to make it optional?

Reply:
done